### PR TITLE
Tidy

### PR DESCRIPTION
**Pull request by glynnbird**
This looks like a lot of change but it is really:

- use standard js linter
- use const/let instead of var
- update dependencies and fix tests
- general spring clean

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```
Comment (garrensmith) [inline .travis.yml]: I think we should bump this higher to 8 and maybe 10 as well

File: app.js [hunk 2]
```diff
@@ -1,101 +1,95 @@
...
+    app.use(bodyParser.json({ limit: '50mb' }))
+    app.use(bodyParser.urlencoded({ extended: false, limit: '50mb' }))
 
     // session support
     if (opts && opts.sessionHandler) {
       app.use(opts.sessionHandler)
     } else {
-      console.log('[OK]  Using default session handler');
-      app.use(session({ 
+      console.log('[OK]  Using default session handler')
```
Comment (garrensmith) [inline app.js]: We should maybe look at adding winston for logging

File: .travis.yml (hunk 1 above)
Comment (glynnbird) [reply .travis.yml]: 👍

File: app.js (hunk 2 above)
Comment (glynnbird) [reply app.js]: 👍

**Pull request merged**